README.md

```diff
@@ -55,6 +55,15 @@ When creating a pull request, please ensure the title conforms to the convention
 
 ---
 
+Turborepo is an intelligent build system optimized for JavaScript and TypeScript codebases. We use Turborepo to facilitate the execution of all our build scripts within the PIE monorepo.
+
+## Remote Caching in AWS S3
+In order to speed up local development / CI workflows, we use Turborepo's remote caching functionality to publish build artifacts to AWS S3. This ensures that only modified packages have their build tasks executed.
+
+In order to take advantage of this functionality, you must set the `TURBO_TOKEN` environment variable on your local machine. Please reach out to the design system team for the value of this token.
```

Comment:
can we give a rough instruction on how to do this for mac + windows?

Reply:
We've opted not to do this in other areas that rely on env vars.

Whilst the steps for setting vars in Windows are standardised, there's many different ways / locations you can set them in MacOS, and usually is down to personal preference.

Reply:
fair point!